### PR TITLE
[README] New Banner

### PR DESCRIPTION
**Pull request by ghost**
the other one is gone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
 <p align="center">
-  <img src="https://cdn.discordapp.com/attachments/472158720399245323/501778941913202708/tkgfrogu.png">
+  <img src="https://user-images.githubusercontent.com/40970965/194487089-f7ff8dd3-331b-42ab-b1c0-54ece2eaebd5.png">
 </p>
-Thanks to Kassin for the meme-worthy banner
 
-## How-to use that mess :
+<p align="center">
+Banner made with 🐸 by <a href="https://github.com/kassindornelles">Kassin Dornelles</a> - Uses frog emojis from the <a href="https://github.com/Riesi/frog_emojis">frog_emojis repository</a> <a href="https://github.com/Riesi/frog_emojis#license">(License)</a> </p>
```
Comment (Riesi) [inline README.md]: a shorter attribution to fulfill CC4 would be nicer 
like this maybe?
```
Uses <a href="https://github.com/Riesi/frog_emojis">frog emojis</a> <a href="https://github.com/Riesi/frog_emojis#license">(License)</a>
```

**Comment by Riesi** [comment]
and maybe add the picture as a file to the repository and link that
because otherwise the banner could get lost again when the link dies :frog:

**Comment by Riesi** [comment]
good job otherwise! I really like the look :frog:

**Comment by Tk-Glitch** [comment]
Looks nice! Good job Kassin :frog: :heart:

**Pull request merged**